texas-hearing-institute/components/Counter.tsx

```diff
@@ -0,0 +1,64 @@
+import {StyleSheet, Text, View} from "react-native";
+import {IndexPath, Select, SelectItem} from "@ui-kitten/components";
+import React, {useEffect, useState} from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+
+
+export default function SyllableCounterDropdown() {
+    const [syllableCount, setSyllableCount] = useState(1);
+    const [selectedIndex, setSelectedIndex] = useState<IndexPath>(new IndexPath(0));
+
+    // Load from storage the first time the page is loading
+    useEffect(() => {
```

Comment:
useEffect is usually used for changes from outside the page - if you have something declared as a useState you should be changing things using the setState methods (and possibly defining custom functions for adding storage functionality) instead of useEffect

Reply:
removed counter.tsx from project